### PR TITLE
Provides stackmap routing within OL. Removes stackmap_base.

### LINKED ISSUE #1044: Move stackmap logic to Orangelight

### PR DESCRIPTION
**Pull request by kevinreiss**
Closes #1044

### CODE REVIEW

File: app/assets/javascripts/availability.js.coffee.erb [hunk 1]
```diff
@@ -243,14 +250,13 @@ class AvailabilityUpdater
     " - #{date_string}"
   title_case = (str) ->
     str[0].toUpperCase() + str[1..str.length - 1].toLowerCase()
-  stackmap_link = (record_id, availability_info, marker_only = false) ->
-    base = "<%= ENV['stackmap_base'] %>"
+  stackmap_link = (record_id, availability_info, marker_only) ->
```
[the remark this answers is not in the document]
Comment (kevinreiss) [reply app/assets/javascripts/availability.js.coffee.erb]: It is used.

File: app/assets/javascripts/availability.js.coffee.erb [hunk 2]
```diff
@@ -66,6 +66,10 @@ class AvailabilityUpdater
         this.get_more_items(holding_id, availability_info['label'])
       else
         this.apply_record_icon(availability_element, availability_info['status'], aeon, availability_info)
+      if availability_info['temp_loc']
+        current_map_link = $("*[data-holding-id='#{holding_id}'] .find-it")
+        temp_map_link = stackmap_link(id, availability_info)
```
Comment (kevinreiss) [inline app/assets/javascripts/availability.js.coffee.erb]: @eliotjordan this was added.

File: app/assets/javascripts/availability.js.coffee.erb [hunk 3]
```diff
@@ -125,7 +129,10 @@ class AvailabilityUpdater
       this.record_needs_more_info(record_id) if availability_info['more_items']
       availability_element = $("*[data-availability-record='true'][data-record-id='#{record_id}'][data-holding-id='#{holding_id}'] .availability-icon")
       if availability_info['temp_loc']
+        current_map_link = $("*[data-location='true'][data-record-id='#{record_id}'][data-holding-id='#{holding_id}'] .find-it")
         $(availability_element).next('.icon-warning').hide()
+        temp_map_link = stackmap_link(record_id, availability_info, true)
```
Comment (kevinreiss) [inline app/assets/javascripts/availability.js.coffee.erb]: @eliotjordan

Comment (eliotjordan) [reply app/assets/javascripts/availability.js.coffee.erb]: Oh. Thanks. Didn't see that addition.

**Pull request merged**